assets/js/main.js

```diff
@@ -34,9 +34,9 @@ define(function() {
     }
 
     //feature list
-    if (hasClass('list-features')) {
+    /*if (hasClass('list-features')) {
         mods.push('mod/list-feature');
-    }
+    }*/
```

Comment:
`Removed features being checked for but not used in this project`

Can we just delete these lines then?

Reply:
Makes sense.